web/src/lib/components/NostrChat.svelte

```diff
@@ -1,307 +1,265 @@
 <script lang="ts">
```

Comment:
This is impossible to review. Please don't change formatting in the same PR as changing other things, to make review easy.